microsoft-365/security/defender-endpoint/linux-exclusions.md

```diff
@@ -49,6 +49,20 @@ Exclusions can be useful to avoid incorrect detections on files or software that
 
 The following table shows the exclusion types supported by Defender for Endpoint on Linux.
 
+### Global Exclusions
+
+Exclusions with scope (`global`) applied at sensor level when real-time protection is enabled.
```

Comment:
This also applies when passive mode is enabled right? If so we should remove the "when real-time protection is enabled"

Reply:
But maybe we can call out that it does not apply in custom scan...

Reply:
Added a Note on linux-global-exclusions.md  "Apart from real-time protection, Global Exclusions are also applicable to passive mode as sensor events will still be passed to EDR. However they will not be applied during a mdatp custom scan."